core_data_modules/cleaners/cleaning_utils.py

```diff
@@ -0,0 +1,79 @@
+import time
+from datetime import datetime
+
+import pytz
+
+from core_data_modules.cleaners import Codes
+from core_data_modules.data_models import Origin, Label
+from core_data_modules.traced_data import Metadata
+
+
+class CleaningUtils(object):
+    @staticmethod
+    def make_cleaner_label(scheme, code, origin_id, origin_name="Pipeline Auto-Coder", date_time_utc=None):
```

Comment:
Consider make_label_from_cleaner

Currently the method on first read makes me think that it's making a cleaner.

Reply:
See the need to change, but would prefer `make_label_from_cleaner_code`, to make it clear that the fn is expecting a code, not a cleaning function, and that the code should have been generated automatically.